catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/ProjectListFragment.java

```diff
@@ -217,7 +227,7 @@ private void importProject(Intent data) {
 						.execute(projectFile);
 			}
 			setShowProgressBar(true);
-		} catch (IOException e) {
```

Comment:
**PRAISE**: The code looks much better than before!

PLEASE-EXPLAIN: What's the source of the null-ptr-exception?

PLEASE-FIX: I think there is a race condition with the adapter. 
The adapter is used in the callback methods, the import method is called "onCreateView", but the adapter is initialized "inOnActivityCreated"  => if the import finishes before the activity was created a null-ptr-exception occurs.
https://stackoverflow.com/questions/28929637/difference-and-uses-of-oncreate-oncreateview-and-onactivitycreated-in-fra

Reply:
Actually I could not reproduce the null pointer exception in testing. But there was a test for it.

Reply:
Ok, which test?
Anyway, please fix the race condition with the adapter (move the code from "onCreateView" to "onActivityCreated").

Reply:
This is the test: org.catrobat.catroid.uiespresso.ui.actionbar.ActionModeMergeTest.onlyTwoProjectsCheckedTest
I moved the code from "onCreateView" to "onActivityCreated", Its much better, the progress dialog is displayed, Thanks!

Reply:
I see. 
Please do not catch the exception instead avoid it.
In most cases, Android Studio tells you where a null-ptr exception can occur. (hover with the mouse over code with yellow background)